tests/testthat/test-cran_downloads.R

```diff
@@ -0,0 +1,15 @@
+context("test-cran_downloads")
+
+test_that("cran_downloads works", {
+  from <- as.Date("2019-02-23", format = "%Y-%m-%d")
```

Comment:
We need to skip this on CRAN, and all the online test cases, actually.

Reply:
that's the only online test actually, now skipped. there was no test for the previous version I think?